PyInstaller/loader/rthooks.dat

```diff
@@ -17,4 +17,5 @@
     'twisted.internet.reactor':        ['pyi_rth_twisted.py'],
     'usb':        ['pyi_rth_usb.py'],
     'win32com':   ['pyi_rth_win32comgenpy.py'],
+    'kivy':       ['pyi_rth_kivy.py', 'pyi_rth_gstreamer.py'],
```

Comment:
It does import it but somehow this was still needed for the rthook to get loaded. Let me double check on this.